Sprint-1/implement/max.js

```diff
@@ -1,4 +1,23 @@
 function findMax(elements) {
+  const numericValues = elements.filter(
+    (element) => typeof element === "number" && !isNaN(element)
+  );
+  let maxNumber = numericValues[0];
+  //console.log(numericValues);
+  if (
+    elements.length === 0 ||
+    elements.every((element) => typeof element != "number")
+  ) {
+    return "-Infinity";
+  }
+
+  for (let i = 1; i < numericValues.length; i++) {
+    if (maxNumber < numericValues[i]) {
+      maxNumber = numericValues[i];
+    }
+  }
+  return maxNumber;
 }
```

Comment:
You could have simplified your algorithm here, I think you were confused about how to represent `-Infinity` in JavaScript.  Here is a simpler algorithm:
```console
let largestSoFar = -Infinity;
for (each element, e,  in the list) {
   if e is numeric and NOT isNaN {
      if (e  > largestSoFar) {
           largestSoFar = e;
      }
   }
}
```